### DISCUSSION TITLE
Is there currently a problem with manifest-validation?

### DISCUSSION THREAD

**Discussion by borque** [microsoft/winget-pkgs#95540] [General] [answered]
Hello!
I wonder whether there is currently a problem with the validation-pipeline since about a week (aproximately since manifest 1.4 was officially introduced)?
I notice an increased rate of "Internal-Error-Dynamic-Scan"-batches, msftbot seems to be out of duty, validation-pipeline is reported unhealthy very often, Feedback-batches are not removed automatically after a reply and there seems to be an unusual high amount of open unapproved PRs.
Don't get me wrong, I don't want to criticize anything. I am just curious because of these observations (which can be completely wrong, of course)!

**Comment by mdanish-kh** [comment, marked as answer]
Yes, the validation pipeline is experiencing issues after the validation for `.zip` was enabled and msftbot is also down as of typing this message. To keep up-to-date with such updates, you can follow the [WinGet's Gitter Channel](https://gitter.im/Microsoft/winget-pkgs)

**Reply by Trenly** [reply]
See also https://github.com/microsoft/winget-pkgs/issues/95543
